exchanges/stream/websocket.go

```diff
@@ -528,6 +529,7 @@ func (w *Websocket) trafficMonitor() {
 					}
 					w.setConnectedStatus(true)
 					trafficTimer.Reset(w.trafficTimeout)
+				default:
```

Comment:
Can we please add a `log.Warnf(log.WebsocketMgr, "Unhandled traffice alert for url: %s", t)`